pages/firebase-outputs.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+
+import {
+  getListOfAllArticlesByAlphabet,
+  getListOfAllArticlesByDate,
+  getListOfLastArticles,
+} from '../firebase/api/articles';
+
+const FirebaseFirst = () => {
+  const [articleList, setArticleList] = useState([]);
+  const [latestArticles, setLatestArticles] = useState([]);
+  const [alphabeticalListOfArticles, setAlphabeticalListOfArticles] = useState(
+    [],
+  );
+
+  useEffect(() => {
+    const listOfArticles = getListOfAllArticlesByDate();
+    listOfArticles.then((value) => setArticleList(value));
+
+    const listOfLastestArticles = getListOfLastArticles();
+    listOfLastestArticles.then((value) => setLatestArticles(value));
+
+    const listOfArticlesAlphabet = getListOfAllArticlesByAlphabet();
+    listOfArticlesAlphabet.then((value) =>
+      setAlphabeticalListOfArticles(value),
+    );
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns

Consider separating + using async/await